negotiation/src/main/java/it/eng/negotiation/model/ContractNegotiationState.java

```diff
@@ -52,7 +52,7 @@ public List<ContractNegotiationState> nextState() {
 	TERMINATED(DSpaceConstants.DSPACE + ContractNegotiationStates.TERMINATED) {
 		@Override
 		public List<ContractNegotiationState> nextState() {
-			return Arrays.asList(TERMINATED);
+			return Arrays.asList();
```

Comment:
This one will cause error if tried to transition to TERMINATED again?
TERMINATED -> TERMINATED?
Is there junit test for it? That canTransit maybe

Reply:
With these new changes it will cause a ContractNegotiationInvalidStateException. We should not allow transitioning to the same state twice. I will add a test for it 👍